SmartPlaces.Facilities/product/Topology/product/src/Topology.csproj

```diff
@@ -0,0 +1,53 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Worker">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<AssemblyName>$(MSBuildProjectName)</AssemblyName>
+		<RootNamespace>$(MSBuildProjectName)</RootNamespace>
+		<CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)..\..\..\..\.config\CodeAnalysis.ruleset</CodeAnalysisRuleSet>
+		<Deterministic>true</Deterministic>
+		<LangVersion>10</LangVersion>
+		<RepositoryRootPath>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\..'))</RepositoryRootPath>
+		<DebugType>Full</DebugType>
+		<EnableNETAnalyzers>true</EnableNETAnalyzers>
+		<AnalysisLevel>latest</AnalysisLevel>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Azure.Extensions.AspNetCore.Configuration.Secrets" Version="1.2.2" />
+		<PackageReference Include="Azure.Identity" Version="1.6.0" />
+		<PackageReference Include="coverlet.collector" Version="3.1.0" />
+		<PackageReference Include="Microsoft.ApplicationInsights.WorkerService" Version="2.21.0" />
+		<PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.8" />
+		<PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.1" />
+		<PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="6.0.0" />
+		<PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="6.0.1" />
+		<PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
+		<PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.1" />
+		<PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
+		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.1" />
+		<PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" />	</ItemGroup>
+
+	<ItemGroup>
+		<ProjectReference Include="..\..\..\..\lib\IngestionManager.Mapped\src\IngestionManager.Mapped.csproj" />
+		<ProjectReference Include="..\..\..\..\lib\IngestionManager\src\IngestionManager.csproj" />
+		<ProjectReference Include="..\..\..\..\lib\OntologyMapper.Mapped\src\OntologyMapper.Mapped.csproj" />
+		<ProjectReference Include="..\..\..\..\lib\OntologyMapper\src\OntologyMapper.csproj" />
+	</ItemGroup>
+
+	<PropertyGroup Condition="'$(BuildingInsideVisualStudio)' != 'true'">
+		<TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+		<MSBuildTreatWarningsAsErrors>True</MSBuildTreatWarningsAsErrors>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<AdditionalFiles Include="$(MSBuildThisFileDirectory)..\..\..\..\.config\rulesets\stylecop.json">
```

Comment:
Are applying rules necessary in this scenario?

Reply:
Style rules?